duktape.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: duktape
+  version: "2.7.0"
+  epoch: 0
+  description: "Embeddable Javascript engine with a focus on portability and compact footprint"
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 90f8d2fa8b5567c6899830ddef2c03f3c27960b11aca222fa17aa7ac613c2890
+      uri: https://github.com/svaarala/duktape/releases/download/v${{package.version}}/duktape-${{package.version}}.tar.xz
+
+  - runs: |
+      DESTDIR=${{targets.destdir}} INSTALL_PREFIX=/usr make -f Makefile.sharedlibrary install
+
+update:
+  enabled: true
+  github:
+    identifier: svaarala/duktape
+    strip-prefix: v
+
+test:
+  pipeline:
+    - uses: test/ldd-check
+      with:
+        packages: ${{package.name}}
```

Comment:
This could also use a pkgconf test:

```
2025-02-21T21:20:09Z WARN 🔗 linter "pkgconf" failed on package "duktape": pkgconfig directory found: usr/lib/pkgconfig/duktape.pc; suggest: This package provides files in a pkgconfig directory, please add the pkgconf test pipeline
```

Reply:
Thanks, done.